src/amber/cli/templates/scaffold/controller/granite/spec/controllers/{{name}}_controller_spec.cr.ecr

```diff
@@ -28,7 +28,6 @@ class <%= class_name %>ControllerTest < GarnetSpec::Controller::Test
     @handler = Amber::Pipe::Pipeline.new
     @handler.build :web do
       plug Amber::Pipe::Error.new
-      plug Amber::Pipe::Logger.new
```

Comment:
The idea here is that the logger is always there now, you can't remove it, correct?

Reply:
We can remove it. I don't see the need for the logger to be on the pipeline for a test.

Reply:
Question: Can logger be turned off for benchmarks, and really fast response times?